src/Symfony/Component/DependencyInjection/Tests/Fixtures/FooBackedEnum.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Tests\Fixtures;
```

Comment:
please add license header (same below)

Reply:
No problem, but have to refork; totally messed up my branches.

One question though: do you think indexing by enum is a good idea? The parameter is instance of `ServiceLocator` and widening `get(string|BackedEnum $id)` is allowed.

Is that correct? If so, this would be allowed:

```php
$locator->get(MyEnum::BAR);
```

Reply:
This wouldn't make sense to me. That's not what enums are for